REQUIRE

```diff
@@ -5,3 +5,5 @@ Distances
 DataFrames
 Clustering
 Colors
+SpatialEcology
```

Comment:
SpatialEcology is small, and only depends on RecipesBase and DataFrames, which you already have - and then on PlotUtils, but I'm happy to drop that dep (it's just to add a custom set of color gradients). So there's not really any cost in practice.